cypress/e2e/shared/simpleTable.test.ts

```diff
@@ -24,258 +24,242 @@ describe('simple table interactions', () => {
   })
 
   it('should render correctly after switching from a dataset with more pages to one with fewer', () => {
-    cy.setFeatureFlags({
-      useGiraffeGraphs: true,
```

Comment:
https://github.com/influxdata/ui/pull/5407/files?w=1

makes this diff much easier to follow